.github/workflows/test_fast.yml

```diff
@@ -62,7 +62,9 @@ jobs:
 
       - name: typing
         run: |
-          mypy
+          mypy --explicit-package-bases
+        # Need --explicit-package-bases because we don't have __init__.py in
+        # top-level dir
```

Comment:
This option will also become available as a config item in `mypy.ini` at some point (it's available on mypy@master but not in the latest 0.812 release)

Reply:
[According to the docs](https://mypy.readthedocs.io/en/stable/command_line.html#cmdoption-mypy-explicit-package-bases) I think that you may need to add `--namespace-packages`

> This option is only useful in conjunction with --namespace-packages.

Reply:
Having said which it seems to work locally...

Reply:
Done but in the mypy config - https://github.com/cylc/cylc-flow/pull/4121/files#diff-6f2d4ba9ca9a357d31014946667b7bed1bfdbc6d2530afc77778fa0a36bee457R8

(where this setting will get migrated in due course)